infermedica_api/models/diagnosis.py

```diff
@@ -117,22 +117,25 @@ def __init__(self, sex, age, time=None, interview_id=None, **kwargs):
     def observations(self):
         return self.symptoms
 
-    def __add_evidence(self, collection, _id, state, time, initial):
+    def __add_evidence(self, collection, _id, state, time, initial=None, source=None):
         """Helper function to update evidence list."""
         evidence = {
             "id": _id,
             "choice_id": state
         }
-
+        if initial is not None:
+            warnings.warn(
+                "Parameter initial is deprecated, please use source with following parameter: initial, suggest, "
+                "predefined, red_flags.", category=DeprecationWarning
```

Comment:
Not sure if above warning is fine.

Reply:
or it should be?: 

> Parameter initial is deprecated, please use `source='initial'` instead.

Reply:
Yes, the second you proposed is much more clear.
But even though it's deprecated we should do `evidence['source'] = 'initial'`

Reply:
@akszydelko done